### PR TITLE
Remove unnecessary attributes `Error` widget.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: docs/error.md [hunk 1]
```diff
@@ -145,21 +145,8 @@ That would generate the following code after validation:
 
 Method | Description | Default
 -------|-------------|---------
+`config(FormModelInterface $formModel, string $attribute, array $attributes = [])` | Configures the widget. |
```
Comment (samdark) [inline docs/error.md:148]: See https://github.com/yiisoft/form/pull/144/files#r745983076

File: src/Widget/Error.php [hunk 2]
```diff
@@ -4,27 +4,52 @@
...
 /**
  * The Error widget displays an error message.
+ *
+ * @psalm-suppress MissingConstructor
  */
 final class Error extends Widget
 {
-    use ModelAttributes;
-
+    private array $attributes = [];
```
Comment (samdark) [inline src/Widget/Error.php:19]: This is confusing. I'd name it `tagAttributes`.

File: src/Widget/Error.php [hunk 3]
```diff
@@ -4,27 +4,52 @@
...
     private string $message = '';
     private array $messageCallback = [];
     private string $tag = 'div';
 
+    /**
+     * Set form interface, attribute name and attributes, and attributes for the widget.
+     *
+     * @param FormModelInterface $formModel Form.
+     * @param string $attribute Form model property this widget is rendered for.
+     * @param array $attributes The HTML attributes for the widget container tag.
```
Comment (samdark) [inline src/Widget/Error.php:32]:
```suggestion
     * Specify a form, its attribute and a list HTML attributes for the error generated.
     *
     * @param FormModelInterface $formModel Form instance.
     * @param string $attribute Form model's property name this widget is rendered for.
     * @param array $attributes HTML attributes for the widget container tag.
```

**Review by samdark — CHANGES_REQUESTED** [review]

File: docs/error.md (hunk 1 above)
Comment (samdark) [reply docs/error.md:148]: But that conflicts with https://github.com/yiisoft/form/pull/146/files#diff-b2dbc04239858c959223583f766058bc1685ade0b8ff4c9278cd3d30866bd5aaR119 :(

File: src/Widget/Error.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Widget/Error.php:19]: https://github.com/terabytesoftw/yii-form/blob/883d98d085c527eb8387f1f2db126029ba0d4c0b/src/Widget/Label.php#L39 those are the global attributes of the widget.

Comment (terabytesoftw) [reply src/Widget/Error.php:19]: done.

**Review by samdark — APPROVED** [review]

**Pull request merged**